paimon-web-ui-new/src/api/models/catalog/index.ts

```diff
@@ -29,4 +29,11 @@ export const getAllCatalogs = () => {
   return httpRequest.get<any, Catalog[]>('/catalog/getAllCatalogs')
 }
 
+/**
+ * # Get database by catalog id
+ */
+export const getDatabaseByCatalogId = (id: number) => {
+  return httpRequest.get<any, Catalog[]>(`/database/getDatabasesById/${id}`)
```

Comment:
Why not `/database/getDatabasesByCatalogId/${id}`? Or `/database/getDatabasesByCondition/${id}`

Reply:
@hunter-cloud09